binaries/fpgad/api/sysfs.c

```diff
@@ -90,7 +91,7 @@ char *cstr_dup(const char *s)
 	char *p;
 	size_t len = strnlen(s, 8192);
 
-	p = malloc(len+1);
```

Comment:
Seeing as we have out own version of cstr_dup, perhaps we should promote it to opae_std and wrap it? opae_str_dup?

Reply:
I'll add opae_strdup().